edenia/antelope-tools
# Feat/improve undiscoverable BPs

### Improve undiscoverable BPs

### What does this PR do?

- Resolves #1131 
- Resolves #1132 
- Resolves #1135 
- Resolves #1136
- Use the right bp.json link in the Rewards Distribution.
- Remove zoom from Rewards Distribution map.

### Steps to test

1. Run the project locally.
2. Go to "Block Producers" page.
3. Scroll down until you find an undiscoverable BP.
4. Click the link "View undiscoverable BPs list".
5. Check the status of the health check of the websites.
6. Check the bp.json url.
7. Click the link "BP JSON Generator".
8. Check the BP JSON Generator completes the candidate name and website.

### Screenshots

**Link to undiscoverable BPs from empty state**
![imagen](https://user-images.githubusercontent.com/66583677/215544219-e8bd3dab-556d-4017-839d-34ed6652408f.png)

**Undiscoverable BPs improvements**
![imagen](https://user-images.githubusercontent.com/66583677/215544430-dc1c4854-40f2-477a-bc7c-4843d7f541ad.png)

**Undiscoverable BPs improvements**
![imagen](https://user-images.githubusercontent.com/66583677/215544392-5d91a6cb-14e1-4b38-a950-aea48c5b774f.png)

**Undiscoverable BPs website health check**
![imagen](https://user-images.githubusercontent.com/66583677/215544466-44eb545a-6322-4911-8463-d6c6839e421a.png)

**Undiscoverable BPs website health check**
![imagen](https://user-images.githubusercontent.com/66583677/215544495-690cc7b2-de14-471d-978b-ab4466f649ec.png)

**Use the right bp.json link instead of the default**
![imagen](https://user-images.githubusercontent.com/66583677/215545859-bf4925a6-8e5e-47fd-b041-2034cd4e56e5.png)


### Checklist
